lib/main/lang/projectService.ts

```diff
@@ -86,6 +86,7 @@ export function quickInfo(query: QuickInfoQuery): Promise<QuickInfoResponse> {
 
 export interface BuildQuery extends FilePathQuery { }
 export interface BuildResponse {
+    project: project.Project;
```

Comment:
Sending the `project` to the foreground is actually quite heavy (its a big tree, that needs to be JSON serialized). Better if you compute something like `tsFilesWithValidEmit` in the background and just send that to the UI process.

Thanks for updating the PR based on reviews :rose: